Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -5399,6 +5399,8 @@ ScrollingNodeID RenderLayerCompositor::updateScrollingNodeForScrollingRole(Rende
             scrollingCoordinator->setScrollingNodeScrollableAreaGeometry(newNodeID, frameView);
             scrollingCoordinator->setFrameScrollingNodeState(newNodeID, frameView);
         }
+        page().chrome().client().ensureScrollbarsController(page(), frameView);
```

Comment:
This is a bit mysterious, and deserves a comment. The commit message doesn't explain why you need it.

Reply:
Fixed.